pkg/backend/httpstate/backend.go

```
@@ -1258,11 +1260,12 @@ func (b *cloudBackend) renderAndSummarizeOutput(
 	close(eventsChannel)
 	<-doneChannel
 
+	permalink := b.getPermalink(update, updateMeta.version, dryRun)
```

Remind me - what is the UX of a DIY (S3 etc) backend but with `--copilot` on?

---

Is silently ignored. All the logic to do something w/ the flag value is in httpstate/backend.go etc.

---

Pattern _seems_ to be to use the logs, so added that.